lib/ansible/cli/galaxy.py

```diff
@@ -28,6 +28,7 @@
 import sys
 import time
 import yaml
+from distutils.version import LooseVersion
```

Comment:
I'd use `from pkg_resources import parse_version`, because `setuptools` is much more up-to-date solution in pythonic world and `distutils` is way to outdated.

Reply:
I used `distutils.LooseVersion` because it's already in use in the galaxy code.

https://github.com/ansible/ansible/blob/v2.5.0/lib/ansible/galaxy/role.py#L230

Reply:
we can always change the version parser later, the current code already uses same